src/Module/tests.jsx

```diff
@@ -129,7 +129,7 @@ describe( 'Module', () =>
                     title        : 'hello',
                 } );
 
-                expect( wrapper.find( IconWithTooltip ).prop( 'onMouseOver' ) )
+                expect( wrapper.find( IconWithTooltip ).prop( 'onMouseEnter' ) )
```

Comment:
The IconWithTooltip-related props here should remain as `onMouseOver`/`onMouseOut`